### ISSUE TITLE
Dark Mode

### ISSUE THREAD

**Issue by curly60e** [LN-Juggernaut/juggernaut-desktop#2]
It will be cool to add a Dark Mode.

**Comment by johncantrell97** [comment]
Yeah, perhaps.  It's clearly all the rage right now and would be pretty easy to do but seems pretty low priority to me.
